config/gadi.config

```diff
@@ -1,38 +1,93 @@
 // profile for the Gadi supercomputer at NCI
-// Before running will need to run: 
-// module load nextflow singularity 
-
-// Gadi specific parameters
 // More info: https://opus.nci.org.au/display/DAE/Nextflow
-params {
-    pbs_account = ''
-    whoami = ''
-}
 
 // Preset use of containers with Singularity
 singularity {
     enabled = true
-    cacheDir = "/scratch/${params.pbs_account}/${params.whoami}/.nextflow/singularity"
+    cacheDir = "/scratch/er01/ndes8648/pipeline_work/nextflow/INFRA-83-Somatic-ShortV/singularity_cache"
```

Comment:
Please revert this to something reproducible by others and refrain from hardcoded paths.

Reply:
Changed to /scratch/${params.gadi_account}/${params.whoami}/singularity